taxi/admin.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib import admin
+
```

Comment:
There is no need to add a blank line between imports from the same group. Avoid it in the future

Reply:
Thank you